src/components/DetailsPanelQuery.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <v-details-panel-header object-type="SQL query" :title="title" />
+    <v-details-panel-list title="Events" :items="object.events" />
+  </div>
+</template>
+
+<script>
+import VDetailsPanelHeader from '@/components/DetailsPanelHeader.vue';
+import VDetailsPanelList from '@/components/DetailsPanelList.vue';
+import { getSqlLabelFromString } from '@/lib/util';
+import { SELECT_OBJECT } from '../store/vsCode';
+
+export default {
+  name: 'v-details-panel-function',
+  components: {
+    VDetailsPanelList,
+    VDetailsPanelHeader,
+  },
+  props: {
+    object: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    title() {
+      return getSqlLabelFromString(this.object.name);
+    },
+
+    events() {
+      return this.$store.state.appMap.events.filter(
+        (e) => e.isCall() && e.codeObject && e.codeObject.id === this.object.id
+      );
+    },
+  },
+
+  mounted() {
```

Comment:
[`created`](https://vuejs.org/v2/api/#created) would be a better choice here. [`mounted`](https://vuejs.org/v2/api/#mounted) is run after the component has been rendered.

Reply:
Done https://github.com/applandinc/appmap-js/pull/132/commits/2a589435914d26694bfed048ec78c74598d04621